Formula/dyld_extractor.rb

```diff
@@ -0,0 +1,15 @@
+class DyldExtractor < Formula
+  desc "Extract dyld shared cache into individual libraries and frameworks"
+  homepage "https://opensource.apple.com/source/dyld/"
+  url "https://opensource.apple.com/tarballs/dyld/dyld-733.6.tar.gz"
+  sha256 "cabbea38a188a3b3c57e3f5ecba2d0124aa5c1edc4676d814ac635410bf1c538"
+
+  depends_on :xcode
```

Comment:
does it actually depend on Xcode tho?

Reply:
yes it does

Reply:
specifically this line: void* handle = dlopen("/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/usr/lib/dsc_extractor.bundle", RTLD_LAZY);

in the section of code trimmed out.

https://opensource.apple.com/source/dyld/dyld-733.8/launch-cache/dsc_extractor.cpp.auto.html